sklearn/utils/_test_common/instance_generator.py

```diff
@@ -304,48 +316,93 @@ def _generate_pipeline():
         )
 
 
+INIT_PARAMS = {
+    SelfTrainingClassifier: dict(estimator=LogisticRegression(C=1)),
+    CalibratedClassifierCV: dict(estimator=LogisticRegression(C=1)),
+    ClassifierChain: dict(base_estimator=LogisticRegression(C=1)),
+    ColumnTransformer: dict(
+        transformers=[
+            ("trans1", StandardScaler(), [0, 1]),
+        ]
+    ),
+    FeatureUnion: dict(transformer_list=[("trans1", StandardScaler())]),
+    FixedThresholdClassifier: dict(estimator=LogisticRegression(C=1)),
+    GridSearchCV: dict(estimator=LogisticRegression(C=1), param_grid={"C": [1.0]}),
+    HalvingGridSearchCV: dict(
+        estimator=Ridge(),
+        min_resources="smallest",
+        param_grid={"alpha": [0.1, 1.0]},
+        random_state=0,
+        cv=2,
+        error_score="raise",
+    ),
+    HalvingRandomSearchCV: dict(
```

Comment:
You probably need to tweak a bit more the parameter for this one to avoid the current failure.

Reply:
This is odd, couldn't reproduce locally with single job, but running tests in parallel I see the failure.

Reply:
Yep this is weird, it looks like a side-effect that should change the state of the random number generator and thus the behaviour (or data).